### ISSUE TITLE
Too many channels and too fast!

### ISSUE THREAD

**Issue by AlbertV97** [uyjulian/spc2it#16]
This toos is very useful but creates 16 channels and i know thats for creating the pan effect but should be an ooption to turn this off, aso the song is hella fast, i think the cmd commands arent too clear of how to change that, please would be great to have an update :D thanks for this awesome tool!

**Comment by uyjulian** [comment]
Please try out VGMTrans.

**Comment by DaVince** [comment]
This is amazing, I had no clue about VGMTrans.

Does this imply that you're done with spc2it, though? You might want to direct people to VGMTrans in the readme and archive the repo if that's the case...

**Comment by cyraid** [comment]
> This is amazing, I had no clue about VGMTrans.
> 
> Does this imply that you're done with spc2it, though? You might want to direct people to VGMTrans in the readme and archive the repo if that's the case...

@DaVince But how can you save as .it files in VGMTrans?

**Comment by uyjulian** [comment]
You don't. You can save as .mid and .sf2

**Comment by AlbertV97** [comment]
Ok, i have learned how to use VGMTrans saving the song sequence and its Intrumentation: sf2 but unfortunately is no compatible with all SPCs for example those from Super Mario World, Jurassic Park and many other SNES games, and spc2it does works with all SPCs no matter what, so i think and update that fixes the speed and an option for making 8 channels only should be really useful and enough for the community that uses this tool a lot, i have successfully made some SPCs out of spc2it, but making it work, was almost like making a song from scratch, because i had to reduce/edit notes and tempo in order to make the conversion, also remove the 8 extra channels that this tool creates, it seems this program was created fot those who likes changing Intrumentation and making mp3 out of SPCs, but this tool is very useful for those who Port music from other games to an specific game, in this case I'm porting music for Súper Metroid, and i really enjoy doing it, so that's why I'd like to try a new update that fixes and making 16 channels can optional, please if anyone have instructions on how to do that on this version I'll appreciate it, if not, and update could be the best, thanks for taking time for reading this petition. 

AlbetV

**Comment by cyraid** [comment]
Yeah I'd rather use a tool that converted to .it directly. :) (as there are effects it seems to miss, doesn't sound right).

**Comment by vince94** [comment]
I agree, MIDI is not the best format to use with SNES music. Broken pitch bends ahoy

**Comment by cyraid** [comment]
*Encourages @uyjulian * :D

**Comment by AlbertV97** [comment]
Honestly i think this tool is pretty impressive how i works, it emulates the effects so great,nit almost does all the hard work for you! And doesn't make SDxx unnecessary comands or Xxx panning comands, that can be made by us ;)

**Comment by uyjulian** [comment]
VGMTrans and spc2it work differently.

VGMTrans knows about the specific format used by each supported game, so it can reduce the speed and channels because it knows it won't be used.

spc2it will write the file as it is playing "streaming", so it does not know about the speed or channels ahead of time. In this case I just made it output at the maximum supported specifications supported by the .it format.
